ApplicationLibCode/ProjectDataModel/RimEclipseCaseTools.cpp

```diff
@@ -81,3 +83,28 @@ std::vector<RimEclipseCase*> RimEclipseCaseTools::allEclipseGridCases()
 
     return {};
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::vector<RimEclipseCase*> RimEclipseCaseTools::allEclipseGridCases()
+{
+    RimProject* proj = RimProject::current();
+    if ( proj )
+    {
+        std::vector<RimEclipseCase*> eclipseCases;
+        for ( auto c : proj->allGridCases() )
+        {
+            if ( dynamic_cast<RimGeoMechCase*>( c ) ) continue;
+
+            if ( auto ec = dynamic_cast<RimEclipseCase*>( c ) )
```

Comment:
No need to handle RimGeoMechCase separately here? It will never match line 100?

Reply:
Yes, thanks.